server/api/include/rs_data_object_finalize.hpp

```diff
@@ -13,12 +13,34 @@ extern "C" {
 
 /// \brief Set the state of every replica for a data object in the catalog, atomically.
 ///
-/// A complete set of the columns in their current state as well as the desired modifications
-/// is required in the form of a JSON structure.
+/// \parblock
+/// The replicas field is a complete set of the columns in their current state as well as the
+/// desired modifications is required in the form of a JSON structure. Each replica can also
```

Comment:
doesn't quite parse as english.

Reply:
Hmmm... I just copied and pasted this from `lib/api/include/data_object_finalize.hpp`. Not sure how that happened. Fixed.